packages/app-store/routing-forms/pages/route-builder/[...appPages].tsx

```diff
@@ -748,7 +748,7 @@ const Routes = ({
   const attributesQueryBuilderConfig = attributes
     ? getQueryBuilderConfigForAttributes({
         attributes: attributes,
-        form: hookForm.getValues(),
+        dynamicOperandFields: hookForm.getValues().fields
```

Comment:
We just needed fields and that makes `getQueryBuilderConfigForAttributes` independent of Routing Form